plugins/code-coverage-backend/README.md

```diff
@@ -0,0 +1,151 @@
+# code-coverage
+
+This is the backend part of the code-coverage plugin. It takes care of processing various coverage formats and standardizing them into a single json format, used by the frontend.
+
+## Configuring your entity
+
+In order to use this plugin, you must set the `backstage.io/code-coverage` annotation.
+
+```yaml
+backstage.io/code-coverage: enabled
+```
+
+There's a feature to only include files that are in VCS in the coverage report, this is helpful to not count generated files for example. To enable this set the `backstage.io/code-coverage` annotation to `scm-only`.
+
+```yaml
+backstage.io/code-coverage: scm-only
+```
+
+Note: This requires the [`backstage.io/source-location` annotation](https://backstage.io/docs/features/software-catalog/well-known-annotations#backstageiosource-location) to be set.
+
+## API
+
+### Adding cobertura report
+
+POST a cobertura xml to `/:kind/:namespace/:name?coverageType=cobertura`
+
+Example:
+
+```bash
+$ cat cobertura.xml | http POST "localhost:7000/api/code-coverage/Component/default/entity-name?coverageType=cobertura" "Content-Type:text/xml"
```

Comment:
`http` needs [`httpie`](https://httpie.io/) to be installed right? Might be worth mentioning since it may not be obvious for people who don't know about it.

Reply:
would recommend changing to curl in docs - way more prevalent and familiar